src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -99,11 +109,17 @@ public static String getReferenceSetID(String url) {
      */
     @Override
     public ReferenceBases getReferenceBases(final PipelineOptions pipelineOptions, final SimpleInterval interval) {
-        Utils.nonNull(pipelineOptions);
```

Comment:
If it's ok for `pipelineOptions` to now be null, document this in the method javadoc. If it's not ok, restore this line.